modular_skyrat/modules/black_mesa/code/ghost_spawners.dm

```diff
@@ -37,6 +37,10 @@
 	prompt_name = "a research facility guard"
 	outfit = /datum/outfit/security_guard
 	you_are_text = "You are a security guard in a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within. DO NOT TRY TO EXPLORE THE LEVEL. STAY AROUND YOUR AREA."
+	flavour_text = "You are a security guard in a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within."
+
+
+
 
 /obj/effect/mob_spawn/ghost_role/human/black_mesa/guard/special(mob/living/carbon/human/spawned_human)
```

Comment:
```suggestion
	flavour_text = "You are a security guard in a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within."

/obj/effect/mob_spawn/ghost_role/human/black_mesa/guard/special(mob/living/carbon/human/spawned_human)
```
Only need 1 line of whitespace

Reply:
i love big whitespace, sorry

Reply:
(So do I, its why I commented it - better to break the habit before it becomes an issue lmao)